lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -464,6 +468,9 @@ <h3 class="error-msgs" id="error-msg"></h3>
      $('#course_price_link').click(function () {
        reset_input_fields();
      });
+     $('#query_registration_code_link').click(function () {
```

Comment:
Legacy code from early last fall. I wouldn't recommend refactoring at this point in time.